tests/acl/test_acl_dscp_change.py

```diff
@@ -0,0 +1,657 @@
+        dut_port = rand_selected_dut.acl_facts()["ansible_facts"]["ansible_acl_facts"]["DATAACL"]["ports"][0]
+    else:
+        dut_port = list(mg_facts['minigraph_portchannels'].keys())[0]
+    if not dut_port:
+        pytest.skip('No portchannels found')
+    if "Ethernet" in dut_port:
+        dut_eth_port = dut_port
+    elif "PortChannel" in dut_port:
+        dut_eth_port = mg_facts["minigraph_portchannels"][dut_port]["members"][0]
+    ptf_src_port = mg_facts["minigraph_ptf_indices"][dut_eth_port]
+
+    topo = tbinfo["topo"]["type"]
+    # Get the list of upstream ports
+    upstream_ports = defaultdict(list)
+    upstream_port_ids = []
+    for interface, neighbor in list(mg_facts["minigraph_neighbors"].items()):
+        port_id = mg_facts["minigraph_ptf_indices"][interface]
+        if (topo == "t1" and "T2" in neighbor["name"]) or (topo == "t0" and "T1" in neighbor["name"]) or \
+                (topo == "m0" and "M1" in neighbor["name"]) or (topo == "mx" and "M0" in neighbor["name"]):
+            upstream_ports[neighbor['namespace']].append(interface)
+            upstream_port_ids.append(port_id)
+
+    return ptf_src_port, upstream_port_ids, dut_port
+
+
+@pytest.fixture(name="setUp", scope="module")
+def fixture_setUp_vxlan_vnet_routes(duthosts,
+                                    rand_one_dut_hostname,
+                                    tbinfo,
+                                    ptfadapter,
+                                    prepare_test_port,
+                                    encap_type):
+    '''
+        Setup for the entire script.
+        The basic steps in VxLAN configs are:
+            1. Configure VxLAN tunnel.
+            2. Configure Vnet and its VNI.
+
+            The testcases are focused on the "configure routes" step. They add,
+            delete, modify, the routes.
+    '''
+    data = {}
+
+    ptf_src_port, ptf_dst_ports, dut_port = prepare_test_port
+    data['ptfadapter'] = ptfadapter
+    data['ptf_src_port'] = ptf_src_port
+    data['ptf_dst_ports'] = ptf_dst_ports
+    data['dut_port'] = dut_port
+
+    duthost = duthosts[rand_one_dut_hostname]
+    minigraph_data = duthost.get_extended_minigraph_facts(tbinfo)
+    data['minigraph_facts'] = minigraph_data
+    data['tbinfo'] = tbinfo
+    data['duthost'] = duthost
+    if data['minigraph_facts']['minigraph_lo_interfaces'][0]['prefixlen'] == 32:
+        data['loopback_v4'] = data['minigraph_facts']['minigraph_lo_interfaces'][0]['addr']
+        data['loopback_v6'] = data['minigraph_facts']['minigraph_lo_interfaces'][1]['addr']
+    else:
+        data['loopback_v4'] = data['minigraph_facts']['minigraph_lo_interfaces'][1]['addr']
+        data['loopback_v6'] = data['minigraph_facts']['minigraph_lo_interfaces'][0]['addr']
+
+    asic_type = duthost.facts["asic_type"]
+    if asic_type in ["cisco-8000", "mellanox"]:
+        data['tolerance'] = 0.03
+    else:
+        raise RuntimeError("Pls update this script for your platform.")
```

Comment:
This will cause test error on Non-Cisco or Non-Mellanox platform. Suggest skipping the test on non-supported platforms.

Reply:
fixed